community/ibm-tomcat-dev/README.md

```diff
@@ -0,0 +1,105 @@
+```
+## PodSecurityPolicy Requirements
+This chart requires a PodSecurityPolicy to be bound to the target namespace prior to installation. Choose predefined ibm-anyuid-psp PodSecurityPolicy.
+
+## Prerequisites
+
+- Kubernetes 1.7+
+- Tiller 2.7.2 or later
+
+## Resources Required
+The chart deploys pods consuming minimum resources as specified in the resources configuration parameter (default: Memory: 200Mi, CPU: 100m)
+
+## Introduction
+
+This chart bootstraps a [Tomcat](https://hub.docker.com/_/tomcat/) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm install --name my-release stable/ibm-tomcat-dev
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Chart Details
+This chart bootstraps a [Tomcat](https://hub.docker.com/_/tomcat/) deployment on a [Kubernetes](http://kubernetes.io) cluster
+
+
+## Configuration
+
+The following table lists the configurable parameters of the Tomcat chart and their default values.
+
+|      Parameter            |          Description            |                         Default                         |
+|---------------------------|---------------------------------|---------------------------------------------------------|
+| `image`                   | The image to pull and run       |  ppc64le/tomcat:8.0                           |
+| `imagePullPolicy`         | Image pull policy               | `Always` if `imageTag` is `latest`, else `IfNotPresent` |
+| `nodeSelector`            | Specify what architecture Node  | `amd64` or `ppc64le`                                    |
+
+
+The above parameters map to `ibm-tomcat-dev` params.
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. 
+
+Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+
+```bash
+$ helm install --name my-release -f values.yaml stable/ibm-tomcat-dev
+```
+
+
+## Support
+
+The helm charts are provided "as-is" and without warranty of any kind.
+
+All helm charts and packages are supported through standard open source forums and helm charts are updated on a best effort basis.
+
+Any issues found can be reported through the links below, and fixes may be proposed/submitted using standard git issues as noted below.
+
+[Submit issue to Helm Chart] ( https://github.com/ppc64le/charts/issues )
+
+[Submit issue to Tomcat docker image]  ( https://github.com/ppc64le/build-scripts/issues )
+
+[Submit issue to Tomcat open source community] (http://tomcat.apache.org/bugreport.html )
+
+
+
+> **Tip**: You can use the default `values.yaml`
+
+## Note (Cluster Image Security)
```

Comment:
Suggesting this approach can break existing workloads in the targeted namespace; instead, we're looking for something closer to [this.](https://github.com/IBM/charts/tree/master/community/portworx#trusted-registries)